pydy/viz/scene.py

```diff
@@ -462,8 +605,8 @@ def remove_static_html(self, force=False):
 
     def display(self):
         """Displays the scene in the default webbrowser."""
-        self.create_static_html()
-        server = Server(scene_file=self._scene_json_file)
+        self._generate_json()
+        server = Server(scene_file=self._scene_json_file, directory=self._static_url)
```

Comment:
I am assigning it here: https://github.com/pydy/pydy/pull/242/files#diff-b43b925b1507940800c850f885764b51R139